namespaces/live.cloud-platform.service.justice.gov.uk/ramya-dev/resources/rds-mysql.tf

```
@@ -0,0 +1,68 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+*/
+module "rds_mysql" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=7.3.0"
```

please update the ref to `9.0.0`
```suggestion
  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=9.0.0"
```